app/build.gradle

```diff
@@ -196,5 +196,5 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
-    buildToolsVersion buildToolsVersion
+    buildToolsVersion '28.0.3'
```

Comment:
Is this change related?

Reply:
No it id not related. I have just updated Build tool version and rest of dependencies to latest available. Mainly work is done in badge.xml

Reply:
Yes please undo it. If you wish to update build tools version then please open a separate issue for it.

Reply:
Yes I have done the changes